niftypet/nipet/prj/src/recon.h

```diff
@@ -8,6 +8,21 @@
 #ifndef RECON_H
 #define RECON_H
 
+/* separable convolution */
+#define KERNEL_RADIUS 5
```

Comment:
radius 5 is wide enough that the edges are at floating point machine precision at 4.5 mm FWHM (<1 sigma)